src/site/content/en/angular/creating-pwa-with-angular-cli/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "Create a Progressive Web App with the Angular CLI"
+subhead: |
+  You want to make your Angular app installable? Wait no more!
+hero: installable.jpg
+alt: Network cables.
+date: 2019-08-15
+description: |
+  Learn how to create installable progressive Angular applications.
+authors:
+  - mgechev
+tags:
+  - angular
+  - pwa
+---
+
+In this post, you'll learn how to use the Angular command line interface (CLI) to make a [Progressive Web App](https://web.dev/discover-installable/) (PWA).
+
+_You can find the code sample from this guide [on GitHub](https://github.com/mgechev/manifest-web-dev)._
+
+{% Aside %}
+
+This post assumes you're already familiar with PWAs and their benefits. If you need a refresher, check out the [Installable](https://web.dev/installable) collection.
+
+{% endAside %}
+
+
+## Create an installable PWA
+
+To make your Angular application a PWA, all you need to do is run a single command!
+
+1. Run this command in the CLI:
```

Comment:
```suggestion
```
Sorry, I didn't catch in the doc that this kind of just repeats the previous sentence. Doesn't really seem like it's adding anything.